pandas/_libs/internals.pyx

```diff
@@ -370,22 +369,16 @@ def get_blkno_indexers(int64_t[:] blknos, bint group=True):
     # There's blkno in this function's name because it's used in block &
     # blockno handling.
     cdef:
-        int64_t cur_blkno
-        Py_ssize_t i, start, stop, n, diff
-
+        Py_ssize_t i, start = 0, stop, n = blknos.shape[0], diff, tot_len
```

Comment:
NOTE, I am adding here a variable definition (```tot_len```)

Reply:
I would put this on multiple lines, this becomes difficult to read IMO